packages/cli/src/commands/hydrogen/init.ts

```diff
@@ -234,7 +234,9 @@ export async function runInit(
       )} to start your local development server and start building`.value,
     ].filter((step): step is string => Boolean(step)),
     reference: [
-      'Building with Hydrogen: https://shopify.dev/docs/custom-storefronts/hydrogen/building/begin-development',
+      'Getting started with Hydrogen: https://shopify.dev/docs/custom-storefronts/hydrogen/building/begin-development',
+      'Hydrogen project structure: https://shopify.dev/docs/custom-storefronts/hydrogen/project-structure',
```

Comment:
This one doesn't show up in your screenshot; does it get filtered out, or is the screenshot outdated?

Reply:
Ah, good catch. I've updated the screenshot now!

Reply:
Added a changeset too 👍🏻